nix/ouroboros-network.nix

```
@@ -43,6 +45,10 @@ let
           doCoverage = config.coverage;
         });
       }
+      {   
+        packages.strict-stm.components.library.configureFlags =
+          lib.mkForce (if checkTVarInvariant then ["-f checktvarinvariant"] else []);
```

Slightly more idiomatic:
```suggestion
          lib.mkIf checkTVarInvariant (lib.mkForce ["-f checktvarinvariant"]);
```